sdk/purview/purview-workflow-rest/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-10-26)
 
 ### Features Added
 
+- Add `/workflows/{id}/validate` API.
+- Add admin query options to workflow / workflow run APIs.
+
 ### Breaking Changes
 
-### Bugs Fixed
+- Split `PurviewWorkflowServiceClient` into multiple clients by model.
```

Comment:
I don't think this description would not work for JS. Because we only have one client.

Reply:
Thanks, updated.